data/plugins/autocomplete.lua

```diff
@@ -698,8 +702,13 @@ end
 ---@param name string
 ---@param character string
 ---@param font? renderer.font
----@param color? string One of the color names from style.syntax
+---@param color? string | renderer.color A style.syntax[] name or specific color
 function autocomplete.add_icon(name, character, font, color)
+  local color_type = type(color)
+  assert(
```

Comment:
Could it be a good idea to also assert if `style.syntax[color]` doesn't exist?

Reply:
done